WIP: Soviet & Udm locales

themohooks opened the pull request.

themohooks committed:
Soviet & Udm locales

themohooks committed:
Update su.strings

veselcraft reviewed (changes requested):
Советский неплох, но удмуртский почему-то не до конца, обрывается на сотых строчках.
Алсо, необходимо указать язык и самого себя в файле `list.yml`, иначе он тупо не будет работать 🤷

themohooks committed:
Fix list.yml, Soviet & Udm fix

Исправление файла list.yml и перевод новых строк на Советский и Удмуртский языки.

themohooks committed:
Update Udm locales

veselcraft reviewed (changes requested):
всё ещё не всё переведено (то что явно на русском, но не отдельные слова)

themohooks committed:
Fix udm strings

themohooks committed:
Update udm strings
